release schema: Add Document.languages, deprecate Document.language (#1380)
open-contracting/standard: docs/history/changelog.md

    * `contracts.maximumValue`
    * `contracts.estimatedValue`
  * [#1372](https://github.com/open-contracting/standard/pull/1372) `Address.country`
  * [#1380](https://github.com/open-contracting/standard/pull/1380) `Document.languages`

* Deprecate some fields:
  * [#1200](https://github.com/open-contracting/standard/pull/1200) `tender.submissionMethod`, because all codes from the `submissionMethod` codelist are deprecated.
  * [#1296](https://github.com/open-contracting/standard/pull/1296) `tender.eligibilityCriteria` in favor of the new `tender.exclusionGrounds` field, in order to use more common terminology and improve semantics. 
  * [#1372](https://github.com/open-contracting/standard/pull/1372) `Address.countryName` in favor of the new `Address.country` field, to promote standardized country codes instead of non-standardized country names.
  * [#1389](https://github.com/open-contracting/standard/pull/1389) `initiationType`, because the approach to data modelling that it supports was not pursued.

>   * [#1380](https://github.com/open-contracting/standard/pull/1380) `Document.language` in favor of the new `Document.languages` field, in order to support documents that contain multiple languages.

Reviewer: ```suggestion
  * [#1380](https://github.com/open-contracting/standard/pull/1380) `Document.language` in favor of the new `Document.languages` field, to support documents in which multiple languages are used.
```
